src/storage/mvcc/txn.rs

```diff
@@ -218,13 +230,23 @@ impl<'a, S: Snapshot + ?Sized> MvccSnapshot<'a, S> {
             }
         }
         // Find the latest write below our start timestamp.
-        match meta.iter_items().find(|x| x.get_commit_ts() <= ts) {
-            Some(x) => {
+        if let Some(x) = first_meta.iter_items().find(|x| x.get_commit_ts() <= ts) {
```

Comment:
@siddontang  Also tried before. It turns out to cause more lines of code. Because the function you mentioned needs to return 3 kinds of results: Ok, Err, and Continue.